### PR TITLE
Add Discord server link

### PR DESCRIPTION
**Pull request by tk3369**
The Humans of Julia is the only Discord server that has a substantial mass of Julia community members. As of June 21, 2021, this server is approaching 1,000 members.

### CODE REVIEW

**Comment by tk3369** [comment]
Cc: @logankilpatrick

**Comment by fredrikekre** [comment]
It's certainly not an open source alternative.

**Comment by tk3369** [comment]
Agree, but that's not a requirement, right?

**Comment by giordano** [comment]
> As an open source alternative to Slack and home to some Julia sub-communities, we also have Zulip and Discord server.

**Comment by tk3369** [comment]
Haha... maybe rephrase it then. I will remove the words "open source" if that's ok.

**Comment by giordano** [comment]
> I will remove the words "open source" if that's ok.

I believe some people appreciate that, so mentioning there is an open source option is valuable. I would rather mention discord separately.

**Review by logankilpatrick — APPROVED** [review]
LGTM unless there are other thoughts

**Comment by tk3369** [comment]
Ready to merge? 

Thanks for the update, Logan.

**Comment by logankilpatrick** [comment]
Merging for now, always open to more discussion on this stuff
